### DISCUSSION TITLE
iklan homepage (halaman utama) shopee

### DISCUSSION THREAD

**Discussion by prasetyodedy** [realodix/AdBlockID#1826] [Personal Filter]
![Screenshot from 2024-08-04 21-43-22](https://github.com/user-attachments/assets/f1e591de-904a-483f-8565-45c162583d4e)

gimana ngehapus semua yang muncul dengan label "Ad"

**Comment by realodix** [comment]
> gimana ngehapus semua yang muncul dengan label "Ad"

Ini menghapus label `Ad` atau hapus kotak yang ada label `Ad`?

Jika hanya label `Ad`, bisa gunakan ini. Tinggal disesuaikan saja untuk memperkecil scope-nya.

```adblock
shopee.co.id##.inline-block:has-text(Ad)
```

kalau ingin kotak yang berlabel `Ad`. Caranya mirip seperti di halaman pencarian ([AdBlockID@a438dc1/src/packages/personal.adfl#L50](https://github.com/realodix/AdBlockID/blob/a438dc134272f7521bbbed3ee2ea55afa5909e3d/src/packages/personal.adfl#L50))

```adblock
shopee.co.id##.section-recommend-products-wrapper .shopee_ic:has(a div[aria-hidden="true"] > .bg-shopee-black26:has-text(/Ad/)):upward(1)
```

Walau bagaimanapun halaman depan cukup cepat perubahannya, harus cukup sering dikaji ulang filternya.

**Reply by prasetyodedy** [reply]
hapus kotak yang berlabel Ad.
terima kasih 🤝
